README.md

```diff
@@ -320,6 +321,17 @@ const inputProps = {
   placeholder: 'Enter city or postcode'
 };
 ```
+<a name="container-props-prop"></a>
+#### containerProps
+
+Provides arbitrary properties to the outer `div` container of Autosuggest. Allows the override of accessibility properties.
+
+```js
+const containerProps = {
+  role: 'button',         // if you use Autosuggest to not allow editing of the input, then the <div> is a button that opens a list
```

Comment:
I'm not sure exactly what this comment means, but also I think it's not necessary - this example doesn't need to go into the details of that, we can just keep it simple :smiley: